packages/techdocs-common/package.json

```diff
@@ -39,6 +39,7 @@
     "@backstage/backend-common": "^0.4.2",
     "@backstage/catalog-model": "^0.6.0",
     "@backstage/config": "^0.1.2",
+    "@backstage/core": "^0.4.3",
```

Comment:
`techdocs-common` is a backend module (used by the techdocs-backend plugin). However `@backstage/core` is a frontend package.
We need the the discovery API that belongs to backend here. :)